src/__tests__/pages/about-us/orbit-blog.test.tsx

```diff
@@ -47,6 +51,23 @@ mockedUseRouter.mockReturnValue({
 })
 
 describe('ORBIT Blog page', () => {
+  const testContext = {
+    query: {
+      page: 1,
+    },
+  } as unknown as GetServerSidePropsContext
```

Comment:
You would think, but TS says no. That's what I tried at first, but going off another example in our codebase (search.test.tsx, article.test.tsx), ended up here to appease the TS overlords.

Reply:
I just ran into this locally too and realized that with out `as unknown` typescript tries to be helpful and let you know it might be a mistake, the way you tell it `no really I do want to do this` is by writing this way. In a test this seems fine to me